bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -750,4 +779,124 @@ mod tests {
         let got = format!("{:#}", n);
         assert_eq!(got, "block-height 100");
     }
+
+    #[test]
+    fn time_from_str_hex_happy_path() {
+        let hex = "0xba70d";
+        let time = Time::from_hex_str(hex).unwrap();
+        assert_eq!(time, Time(763_661));
```

Comment:
Nit: would be easier to review if this was hex literal too. :)

Reply:
resolved for all tests, just using a const now for each set of tests